src/core/client/admin/components/UserHistoryDrawer/UserHistoryDrawerContainer.css

```diff
@@ -1,3 +1,8 @@
+$user-drawer-background: var(--v2-colors-pure-white);
+$user-drawer-account-info: var(--v2-colors-mono-500);
+$user-drawer-divider: var(--v2-colors-grey-300);
+$user-drawer-status-label: var(--v2-colors-mono-500);
+
 .comments {
   flex: auto;
   overflow: scroll;
```

Comment:
<img width="614" alt="image" src="https://user-images.githubusercontent.com/5751504/68170877-b57da680-ff2e-11e9-8318-5045570246c0.png">

I'm seeing an extra scroll bar here, `overflow: auto` seems to fix it.